sdk/monitor/monitor-opentelemetry-exporter/samples-dev/BasicTracerNode.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit, let's stick with camelCase names for the files: `basicTracerNode.ts`

This should also apply to any files in utils unless they only export a single object of the same name. For example, I generally expect a file named `utils/Tracer.ts` to either `export class Tracer` and nothing else, or `export default class Tracer`.